### PR TITLE
Port `r` (resize) commands to the rzshell

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- Migrate `r` (resize) commands to the rzshell
- Removed `rizin`, `rz-bin`, etc commands to call tools from inside the Rizin
- Removed `reset` command to reset the terminal (Not sure about usability of such command, let me know if I should return it)
- Removed `rm $<name>` command to remove alias, since I think it's better to have it in the `$` command, let me know if I should restore it.

**Test plan**

CI is green

Partially addresses #1342

### CODE REVIEW

File: librz/core/cmd_descs/cmd_resize.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    cname: resize_insert
+    summary: Insert num bytes, move following data up
+    args:
+      - name: num
+        type: RZ_CMD_ARG_TYPE_RZNUM
+  - name: rb
+    cname: rebase
+    summary: Rebase all flags, binary information, breakpoints, and analysis
+    args:
+      - name: oldbase
```
Comment (ret2libc) [inline librz/core/cmd_descs/cmd_resize.yaml:33]:
```suggestion
      - name: newbase
```
???

File: librz/core/cmd_resize.c [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+static RzCmdStatus resize_helper(RzCore *core, st64 delta) {
+	if (!core->file) {
+		return RZ_CMD_STATUS_ERROR;
+	}
+	ut64 oldsize = (core->file) ? rz_io_fd_size(core->io, core->file->fd) : 0;
+	if (oldsize == -1) {
+		return RZ_CMD_STATUS_ERROR;
+	}
```
Comment (ret2libc) [inline librz/core/cmd_resize.c:31]: can it be removed?

**Review by ret2libc — COMMENTED** [review]
LGTM. just minor stuff

File: librz/core/cmd_resize.c (hunk 2 above)
Comment (XVilka) [reply librz/core/cmd_resize.c:31]: The `oldsize` check?

Comment (ret2libc) [reply librz/core/cmd_resize.c:31]: I mean ... Why is it needed?

Comment (XVilka) [reply librz/core/cmd_resize.c:31]: It was in the old code.

File: librz/core/cmd_descs/cmd_resize.yaml (hunk 1 above)
Comment (XVilka) [reply librz/core/cmd_descs/cmd_resize.yaml:33]: This is how old syntax of the `rb` worked before, I did not change the semantics

Comment (ret2libc) [reply librz/core/cmd_descs/cmd_resize.yaml:33]: Mmmm... ok, let's keep this for now but it doesn't make sense at all imo :/ the oldbase should be known by rizin itself, you don't need users to tell you the oldbase.

**Review by ret2libc — APPROVED** [review]

**Pull request merged**